README.md

```diff
@@ -1,3 +1,7 @@
+[![Integration](https://github.com/opensearch-project/opensearch-ruby/actions/workflows/main.yml/badge.svg)](https://github.com/opensearch-project/opensearch-ruby/actions/workflows/main.yml)
```

Comment:
Should this include the branch name? 

https://github.com/opensearch-project/opensearch-ruby/actions/workflows/main.yml/badge.svg?branch=main

I would rename this to just "Tests" :)

Reply:
@dblock if no branch is specified, i believe it picks default branch from settings. Do you recommend to add branch explicitly?